.github/workflows/image-builder.yaml

```diff
@@ -2,24 +2,35 @@ name: image-builder
 
 on:
   schedule:
-  - cron: "0 6 * * Sun"
+  - cron: "0 5 1 * *"
 jobs:
   image-builder:
     runs-on: [self-hosted, linux]
     steps:
     - name: Checkout image-builder
       uses: actions/checkout@v3
       with:
-        repository: 'kubernetes-sigs/image-builder'
-        ref: master
+        repository: 'outscale-vbr/image-builder'
+        path: "add-outscale-image"
+        ref: add-outscale-image
```

Comment:
Any reason to go back to our fork ?

Reply:
https://github.com/outscale-vbr/image-builder/pull/2/files
Yes because packer use now outscale instead of osc.
I have to add a new PR to image builder.

Reply:
When those PR (and also the one with  global permission ) will be accept, review, merged, we will switch back to the official repo.